acceptance-tests/features/support/config.rb

```diff
@@ -25,3 +25,6 @@
 
 ### Require gems such as httprb for interacting with the APIs
 require 'http'
+
+### Require pry gem for debugging purposes
+require 'pry'
```

Comment:
Yeh I kinda left that in as I got sick of writing require 'pry' binding.pry every time I wanted to use it =) But good point about the fact it shouldn't really be in there.